scripts/apps/search/components/fields/scheduledDateTime.tsx

```diff
@@ -14,15 +15,23 @@ class ScheduledDateTime extends React.PureComponent<IProps> {
     }
     render() {
         const scheduled = this.props.item.archive_item
-            ? this.props.item.archive_item.schedule_settings.utc_publish_schedule : null;
+            ? this.props.item.archive_item.schedule_settings.utc_publish_schedule
+            : appConfig?.features?.showPublishSchedule ? this.props.item?.publish_schedule : null;
```

Comment:
actually not sure why `scheduled` which seems to be the value should depend on the feature config